src/app/core/data/remote-data-error.ts

```diff
@@ -1,6 +1,6 @@
 export class RemoteDataError {
   constructor(
-    public statusCode: string,
+    public statusText: string,
```

Comment:
I realize I forgot to mention... in testing this PR yesterday, I discovered that this `RemoteDataError` change breaks the `search-results` component on `master`. Unfortunately, it currently depends on / uses the old `statusCode` property here: https://github.com/DSpace/dspace-angular/blob/master/src/app/+search-page/search-results/search-results.component.html#L10-L11

Hopefully we can find a way to resolve this, but this is the cause of (at least some) Search Results problems